hub/api/tests/test_api.py

```diff
@@ -447,33 +443,6 @@ def test_fails_on_wrong_tensor_syntax(memory_ds):
     memory_ds.some_tensor = np.ones((28, 28))
 
 
-@pytest.mark.skipif(not has_hub_testing_creds(), reason="requires hub credentials")
-def test_hub_cloud_dataset():
-    username = "testingacc"
-    password = os.getenv("ACTIVELOOP_HUB_PASSWORD")
-    id = str(uuid.uuid1())
-
-    uri = f"hub://{username}/hub2ds2_{id}"
-
-    client = HubBackendClient()
-    token = client.request_auth_token(username, password)
-
-    with Dataset(uri, token=token) as ds:
-        ds.create_tensor("image")
-        ds.create_tensor("label", htype="class_label")
-
-        for i in range(1000):
-            ds.image.append(i * np.ones((100, 100)))
-            ds.label.append(np.uint32(i))
-
-    ds = Dataset(uri, token=token)
-    for i in range(1000):
-        np.testing.assert_array_equal(ds.image[i].numpy(), i * np.ones((100, 100)))
-        np.testing.assert_array_equal(ds.label[i].numpy(), np.uint32(i))
-
-    ds.delete()
-
-
```

Comment:
just noticed, test_array interface is comparing arr1 to arr1 instead of arr2. changing to arr2 makes the test fail which is weird.

Reply:
i don't understand, should i change something? is there a broken test?

Reply:
See Line 454
`assert arr1.__array_interface__["data"][0] == arr1.__array_interface__["data"][0]`

Reply:
why do we even check the __array_interface__?

Reply:
ohh you know what, it's probably because @farizrahman4u had this test in here when he was caching the arrays, but since it is arr1 comparing with arr1 it didn't fail

Reply:
yup, you can add a TODO and a task in the backlog if the test fails and you think it's out of scope.